static/bidder-info/dianomi.yaml

```diff
@@ -16,7 +16,11 @@ userSync:
   iframe:
     url: 'https://www-prebid.dianomi.com/prebid/usersync/index.html?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}'
     userMacro: '$UID'
+  redirect:
+    url: 'https://data.dianomi.com/frontend/usync?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}'
+    userMacro: '$UID'
   # Dianomi supports user syncing, but requires configuration by the host. contact this
   # bidder directly at the email address in this file to ask about enabling user sync.
   supports:
```

Comment:
Please remove the `supports` section entirely. This is only intended to be used when there is no user sync configured by default.

Reply:
Done 👍